manage.py

```diff
@@ -240,18 +240,27 @@ def add(ctx, collections, note, name, tables_only, field_counts_option, field_li
         name = f"collection_{'_'.join(str(_id) for _id in sorted(collections))}"
 
     schema = f'view_data_{name}'
+
+    db.execute('CREATE SCHEMA IF NOT EXISTS summaries')
+    db.set_search_path(["summaries"])
+    db.execute('''CREATE TABLE IF NOT EXISTS selected_collections
+                  (id SERIAL, schema TEXT NOT NULL, collection_id INTEGER NOT NULL)''')
+    db.execute('CREATE UNIQUE INDEX IF NOT EXISTS selected_collections_id ON selected_collections (id)')
+    db.execute("""CREATE UNIQUE INDEX IF NOT EXISTS selected_collections_schema_collection_id
+                  ON selected_collections (schema, collection_id)""")
+    db.execute_values('INSERT INTO selected_collections (schema, collection_id) VALUES %s',
+                      [(schema, _id,) for _id in collections])
```

Comment:
The quoting style is inconsistent. Add a newline before the INSERT as it is logically separate (tool setup versus collection summarization).